docs/Secure-Coding-Guide-for-Python/CWE-703/CWE-754/noncompliant01.py

```diff
@@ -1,19 +1,35 @@
 # SPDX-FileCopyrightText: OpenSSF project contributors
 # SPDX-License-Identifier: MIT
-""" Non-compliant Code Example """
+"""Non-compliant Code Example"""
+
 import sys
 
 
 class Package:
+    """Class representing a package object"""
+
     def __init__(self):
-        self.package_weight = float(1.0)
+        self.package_weight = float(0.0)
+        self.max_package_weight = 100.0
```

Comment:
Same suggestion as in README:
```suggestion
        self.package_weight: float = 0.0
        self.max_package_weight: float = 100.0
```